### DISCUSSION TITLE
how to change color used for pinyin tone indication?

### DISCUSSION THREAD

**Discussion by MarkoSchuetz** [xiaoyifang/goldendict-ng#1269] [Q&A]
I've been using TOFU learn from a while to memorize Chinese characters and words. So, I'm used to the coloring from that app. I'm also using GoldenDict on my laptop, but the colors it uses are different. This is actually very confusing.
| tone | TOFU | GoldenDict |
|--------|----------|------------------|
| 1.  | red | red|
| 2. | green | orange |
| 3. | blue | green |
| 4. | pink |  blue |

Since I'm already used to the colors in TOFU learn, I'd like to change the colors in GoldenDict to match.

How would I go about doing this?

**Comment by xiaoyifang** [comment]
I think this is dictionary related.

you can change the dictionary's style using the dictionary's own css .
